src/HeroIcon/HeroIcon.js

```diff
@@ -0,0 +1,118 @@
+/* eslint-disable import/no-unresolved */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { dcnb } from 'cnbuilder';
+import {
+  VideoCameraIcon,
+  MicrophoneIcon,
+  LockClosedIcon,
+} from '@heroicons/react/outline';
+import {
+  ArrowRightIcon,
+  ChevronDownIcon,
+  ChevronRightIcon,
+  DownloadIcon,
+  MailIcon,
+  PlayIcon,
+} from '@heroicons/react/solid';
```

Comment:
@rebeccahongsf Looks like you might need to import the solid lock closed icon too since you reference it in the local footer